### PR TITLE
[11.x] ISPN-12506 Fix protobuf schema lack of permission

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-12506

### CODE REVIEW

**Comment by pruivo** [comment]
@karesti I'm unable to test it. I got a blank page even with authentication disabled. better ask someone else.

File: src/services/protobufService.ts [hunk 1]
```diff
@@ -115,58 +81,89 @@ class ProtobufService {
...
+      actionResponse = <ActionResponse>{
+        message: err.message,
+        success: false,
+      };
+    } else if (err instanceof Response) {
+      if (err.status.valueOf() == 403) {
+        // Not Found
+        actionResponse = <ActionResponse>{
+          message:
+            'Unauthorized access. The user lacks ___schema_manager permission.',
```
Comment (pruivo) [inline src/services/protobufService.ts:153]:
```suggestion
            'Unauthorized access.',
```
I don't think it makes sense to leak the permission required. The permission's name is going to change and, using other sites as an example, they just say "unauthorized".

**Comment by pruivo** [comment]
nitpicking... if it isn't too much trouble, maybe "hide" the "Create protobuf schema" if not authorized (or the full tab)?

**Comment by karesti** [comment]
> nitpicking... if it isn't too much trouble, maybe "hide" the "Create protobuf schema" if not authorized (or the full tab)?

Yes. this will be part of a higher auth related feature. We need to do the bare minim here, like we do with other stuff just showing the error. but not dealing with having or not permissions. a well done feature is coming for that

**Comment by pruivo** [comment]
integrated! thanks @karesti !

**Pull request merged**